Source: Written by Thomas Anderson
Upstream: Unknown
Reason: Make the tests optional, so they are only run when the 
test target is called

--- a52dec-0.7.4-orig/test/Makefile.am	2001-12-16 01:44:28.000000000 +0000
+++ a52dec-0.7.4/test/Makefile.am	2008-08-22 14:56:01.000000000 +0000
@@ -1,4 +1,4 @@
-noinst_PROGRAMS = compare
+check_PROGRAMS = compare
 compare_SOURCES = compare.c
 compare_LDADD = -lm
 
